src/api/MoneyMoovApiClient.ts

```diff
@@ -0,0 +1,73 @@
+import { deletePaymentRequest, getPaymentRequests, voidPaymentRequest } from "./PaymentRequestClient";
+import { PaymentRequestPageResponse } from "./types/ApiResponses";
+import { ApiError } from "./types/ApiResponses";
+
+class MoneyMoovApiClient {
+  apiUrl: string;
+  token: string;
+
+  /**
+   * @param apiUrl The api base url
+   * @param token The token
+   */
+  constructor(apiUrl: string, token: string) {
+    this.apiUrl = apiUrl;
+    this.token = token;

```

Comment:
What do you think of making the token the first argument so `apiUrl` could be optional and default it to the prod api url?

It might not be much but will mean one parameter less for them?